app/src/main/java/com/sunasterisk/travelapp/data/source/network/NetworkError.kt

```diff
@@ -0,0 +1,27 @@
+package com.sunasterisk.travelapp.data.source.network
+
+import java.io.IOException
+
+class NetworkError : Throwable {
+    private val error: Throwable?
+
+    constructor(e: Throwable) : super(e) {
+        error = e
+    }
```

Comment:
viết constructor kiểu kotlin 
```suggestion
class NetworkError(e: Throwable) : Throwable(e) {
```